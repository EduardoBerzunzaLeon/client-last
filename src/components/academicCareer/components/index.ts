export * from './StudentCard';
export * from './UserCreatorCard';
