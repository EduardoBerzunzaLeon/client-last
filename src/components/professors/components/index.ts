export * from './ButtonProfessorActive';
export * from './columns';
export * from './course';
export * from './ExcelButtonProfessors';
export * from './Header';
export * from './ProfessorDataForm';
export * from './ProfessorDialog';
