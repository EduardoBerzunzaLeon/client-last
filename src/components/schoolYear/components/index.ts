export * from './PhaseCard';
export * from './FileErrorsTable';
export * from './CurrentSubjectsForm';
export * from './FailedSubjectsForm';
export * from './ValidationPasswordForm';
export * from './SchoolYearStepForm';
