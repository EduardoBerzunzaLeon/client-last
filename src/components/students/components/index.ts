export * from './columns';
export * from './Header';
export * from './StudentDataForm';
export * from './StudentDialog';
export { ProfessorsHistoryDialog } from './professorsHistory';
