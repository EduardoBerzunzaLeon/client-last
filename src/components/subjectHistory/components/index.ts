export * from './SubjectsStudied';
export * from './CurrentSubjects';
export * from './UnstudySubjects';
export * from './columns';
export * from './PhaseDialog';
export * from './Header';
export * from './PhaseDataForm';
