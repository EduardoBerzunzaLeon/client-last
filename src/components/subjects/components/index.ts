export * from './Header';
export * from './columns';
export * from './SubjectDialog';
export * from './SubjectsList';
export * from './SubjectDetailDialog';
