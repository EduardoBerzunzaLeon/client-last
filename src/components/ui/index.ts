export * from './badge/Badge';
export * from './divider/Divider';
export * from './errorCard/ErrorCard';
export * from './excelButton/ExcelButton';
export * from './headerAdmin/HeaderAdmin';
export * from './skeleton';
export * from './skeletonDropdown/SkeletonDropdown';
export * from './slipButton/SlipButton';
export * from './spanDetail/SpanDetail';
export * from './spinner/Spinner';
export * from './spinnerRTK/SpinnerRTK';
