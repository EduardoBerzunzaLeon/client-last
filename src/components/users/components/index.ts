export * from './AdminPasswordForm';
export * from './ButtonUserBlocked';
export * from './columns';
export * from './Header';
export * from './UserDataForm';
export * from './UserDialog';
