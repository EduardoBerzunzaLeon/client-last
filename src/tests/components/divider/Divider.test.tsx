import { shallow } from 'enzyme';
import { Divider } from '../../../components/ui';

describe('<Divider />', () => {
  test('should match to snapshot', () => {
    const wrapper = shallow(<Divider text="Name" icon="user" />);
    expect(wrapper).toMatchSnapshot();
    expect(wrapper.find('i').prop('className')).toContain('pi-user');
    expect(wrapper.find('b').text().trim()).toContain('Name');
  });
});
