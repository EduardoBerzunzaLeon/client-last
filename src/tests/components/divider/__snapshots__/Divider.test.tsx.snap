// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Divider /> should match to snapshot 1`] = `
<Divider
  align="left"
  className={null}
  layout="horizontal"
  style={null}
  type="solid"
>
  <div
    className="inline-flex align-items-center text-purple-700 "
  >
    <i
      className="pi pi-user mr-2"
    />
    <b>
      Name
    </b>
  </div>
</Divider>
`;
